### PR TITLE
fix(kit): `InputDate*`-components' filler with custom date-separator

### LINKED ISSUE #1172: 🐞 - InputDate filler (placeholder) is still dd.mm.yyyy when setting TUI_DATE_SEPARATOR to '/'
### Which @taiga-ui/* package(s) are the source of the bug?

kit

### Please provide a link to a minimal reproduction of the bug

https://stackblitz.com/edit/angular-zsf6pv

### Description

When using
```
{
  provide: TUI_DATE_SEPARATOR,
  useValue: '/',
}
```
the filler (placeholder) of `InputDate` is still `dd.mm.yyyy`. I would expect it to be `dd/mm/yyyy`.

When typing a date, the separator is correctly set to `/`.

This leads to this weird situation:
Screenshot 2022-01-05 at 21 34 03

### Angular version

12.2.15 / 13.1.1

### Taiga UI version

2.29.0

### Which browsers have you used?

- [X] Chrome
- [X] Firefox
- [ ] Safari
- [x] Edge

### Which operating systems have you used?

- [X] macOS
- [ ] Windows
- [ ] Linux
- [ ] iOS
- [ ] Android

### PR DESCRIPTION
**Pull request by nsbarsukov**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [X] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/)
- [X] Tests for the changes have been added (for bug fixes / features)
- [X] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [X] Bugfix
- [ ] Feature
- [ ] Refactoring
- [ ] Code style update
- [ ] Build or CI related changes
- [ ] Documentation content changes

## What is the current behavior?

Closes #1172

## Does this PR introduce a breaking change?

- [ ] Yes
- [X] No

### CODE REVIEW

File: projects/kit/components/input-date-range/input-date-range.component.ts [hunk 1]
```diff
@@ -134,7 +135,10 @@ export class TuiInputDateRangeComponent
     open = false;
 
     readonly maxLengthMapper: TuiMapper<TuiDay, TuiDay> = MAX_DAY_RANGE_LENGTH_MAPPER;
-    readonly dateFiller$ = this.dateTexts$.pipe(pluck(this.dateFormat));
+    readonly dateFiller$ = this.dateTexts$.pipe(
+        pluck(this.dateFormat),
```
Comment (waterplea) [inline projects/kit/components/input-date-range/input-date-range.component.ts:139]: `pluck` is deprecated. If we introducing a `map` anyway, let's add extracting in it as well and drop `pluck`.

File: projects/kit/components/input-date-time/input-date-time.component.ts [hunk 2]
```diff
@@ -111,7 +112,10 @@ export class TuiInputDateTimeComponent
 
     open = false;
     readonly filler$ = combineLatest([
-        this.dateTexts$.pipe(pluck(this.dateFormat)),
+        this.dateTexts$.pipe(
+            pluck(this.dateFormat),
+            map(dateFiller => changeDateSeparator(dateFiller, this.dateSeparator)),
```
Comment (waterplea) [inline projects/kit/components/input-date-time/input-date-time.component.ts:117]: Same

File: projects/kit/components/input-date/input-date.component.ts [hunk 3]
```diff
@@ -120,7 +121,10 @@ export class TuiInputDateComponent
     defaultActiveYearMonth = TuiMonth.currentLocal();
 
     open = false;
-    readonly filler$ = this.dateTexts$.pipe(pluck(this.dateFormat));
+    readonly filler$ = this.dateTexts$.pipe(
+        pluck(this.dateFormat),
+        map(dateFiller => changeDateSeparator(dateFiller, this.dateSeparator)),
```
Comment (waterplea) [inline projects/kit/components/input-date/input-date.component.ts:126]: And here

File: projects/kit/components/input-date-range/input-date-range.component.ts (hunk 1 above)
Comment (nsbarsukov) [reply projects/kit/components/input-date-range/input-date-range.component.ts]: fixed

**Review by waterplea — APPROVED** [review]

**Review by vladimirpotekhin — APPROVED** [review]

**Comment by pwespi** [comment]
Thanks  for the fast fix! Great job! 🚀

**Pull request merged**